mlperf_logging/rcp_checker/1.0.0/rcps_resnet.json

```diff
@@ -50,11 +50,11 @@
       "epsilon": 0,
       "opt_learning_rate_warmup_epochs": 5,
       "opt_momentum": 0.9,
-      "opt_weight_decay": 2e-3,
-      "opt_learning_rate_decay_steps": 6720
+      "opt_weight_decay": 2e-4,
+      "opt_learning_rate_decay_steps": 6095
     },
     "Epochs to converge": [
-      41, 40, 42, 42, 41, 41, 42, 42, 41, 41]
+      42, 44, 43, 41, 41, 41, 42, 42, 43, 41]
```

Comment:
I'm a bit confused about this change. Isn't the average of `42, 44, 43, 41` `43` larger than the average of `41, 40, 42, 42` `41`? So, if `"opt_weight_decay": 2e-3, "opt_learning_rate_decay_steps": 6720` converges faster, why are we replacing it with a slower set of hyper-parameters `"opt_weight_decay": 2e-4, "opt_learning_rate_decay_steps": 6095`?

Reply:
Shang, I am not running these experiments, I am just getting the numbers and adding them to the json files.

Reply:
Are you saying the previous RCPs (i.e., `[41, 40, 42, 42, 41, 41, 42, 42, 41, 41]`) are invalid or contain mistakes? Sorry I might be missing some context here.

Reply:
For some reason these numbers were updated by Yuechao -- probably he did another run and fixed something. But these are the up-to-date ones.

Reply:
@sgpyc I'm wondering if you could elaborate on what's going on here?

Reply:
FYI:

Most recent resnet RCP spreadsheet https://docs.google.com/spreadsheets/d/1CDCSizkWfJKIR0pS4E1k36gNuWVVEORCfIMxM2U-PDQ/edit?resourcekey=0-nOaUKM7iS1jvI0CJmPtMSw#gid=1086699069

Most recent Bert spreadsheet
https://docs.google.com/spreadsheets/d/1vD3UkqJr20dB1jOoZGuLmIbf5iFgsk1aObTMC2QhvSY/edit#gid=252780832

Reply:
@sgpyc  could you please explain why we're going with slower RCPs now?

Reply:
> FYI:
> 
> Most recent resnet RCP spreadsheet https://docs.google.com/spreadsheets/d/1CDCSizkWfJKIR0pS4E1k36gNuWVVEORCfIMxM2U-PDQ/edit?resourcekey=0-nOaUKM7iS1jvI0CJmPtMSw#gid=1086699069
> 
> Most recent Bert spreadsheet
> https://docs.google.com/spreadsheets/d/1vD3UkqJr20dB1jOoZGuLmIbf5iFgsk1aObTMC2QhvSY/edit#gid=252780832

I can't access those google docs

Reply:
Sorry for the confusion.
I mis-read the decay_steps from v0.7's logs (some submitters included the warmup epochs, and others didn't), and found out that while running the 64k experiment. The previous 8k RCP was short of 5 epochs, and its chance of converge was border line. I tried using the two 8k HP sets from v0.7 with the correct number of train epochs, and the one in this PR comes out better. That's the reason of the 8k RCP update.

BTW, the "decay_steps" listed in rcps_resnet.json still follows what v0.7 submissions reported: 32k & 4k's decay_steps included the warmup epochs, but 64k, 8k and 2k's didn't. Should update them if we know which of the ways to pick; otherwise, should put a note there.

@shangw-nvidia : I have shared the two RCP tables with training@mlcommons.org. If still having access issues, please request access on the error page, and I will share them with the account you are using.